rvm-builder/run-builder.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+USAGE="./run-builder.sh node-name s3://your-bucket"
+NODE_NAME=$1
+BUCKET_NAME=$2
+salt_parms="cwd=/home/$USER"
+
+if [ -z "${NODE_NAME}" ] || [ -z "${BUCKET_NAME}" ]; then
+    echo "Usage: $USAGE"
+    exit 1
+fi
+
+echo "Copying over scripts needed to setup vm, rvm, and install ruby..."
+scp builder-scripts/*.sh $NODE_NAME:~/
```

Comment:
Do we want to do something like ```
RVM_BUILDER_FOLDER=`dirname "$0"`

scp $RVM_BUILDER_FOLDER/builder-scripts/*.sh $NODE_NAME:~/
```

So that this script can be run from any dir?